### PR TITLE
fix(vpnx-rust): licenses data generation

### PR DESCRIPTION
**Pull request by doums**
fix rust build pipeline

### CODE REVIEW

File: nym-vpn-x/src-tauri/about.toml [hunk 1]
```diff
@@ -3,6 +3,7 @@ accepted = [
     "Apache-2.0 WITH LLVM-exception",
     "BSD-2-Clause",
     "BSD-3-Clause",
+    "BSL-1.0",
```
Comment (octol) [inline nym-vpn-x/src-tauri/about.toml:6]: Is this https://en.wikipedia.org/wiki/Business_Source_License ? Let me quickly look into it

**Review by octol — CHANGES_REQUESTED** [review]

**Review by zaneschepke — APPROVED** [review]

File: nym-vpn-x/src-tauri/about.toml (hunk 1 above)
Comment (octol) [reply nym-vpn-x/src-tauri/about.toml:6]: What code is using? As far as I can tell it's not compatible with GPL-3.0?

Comment (doums) [reply nym-vpn-x/src-tauri/about.toml:6]: @octol I think it's https://opensource.org/license/bsl-1-0

Comment (octol) [reply nym-vpn-x/src-tauri/about.toml:6]: Ah good

**Review by octol — APPROVED** [review]

**Pull request merged**